2.Lets-Build-OTPublisher/Lets-Build-OTPublisher/TBExampleVideoCapture.m

```diff
@@ -644,7 +644,7 @@ - (BOOL)imageBufferIsSanitary:(CVImageBufferRef)imageBuffer
         
         if (hasNextAddress) {
             size_t planeLength =
-            dataWidth * CVPixelBufferGetHeightOfPlane(imageBuffer, i);
```

Comment:
Oh, I remember this. AV Foundation was giving us some bad values and I was scrambling to write a "sane" continuity checker. Definitely don't want `width * height ^ 2`; thanks for the fix.